Gemfile.lock

```diff
@@ -219,9 +218,18 @@ GEM
     responders (2.4.0)
       actionpack (>= 4.2.0, < 5.3)
       railties (>= 4.2.0, < 5.3)
-    rsolr (2.0.1)
-      builder (>= 2.1.2)
-      faraday
+    rspec-core (2.14.8)
```

Comment:
Ya, you need to run `bundle install` and then commit the changes to `Gemfile.lock` so that these disappear